org.OpenT2T.Sample.SuperPopular.Thermostat/org.OpenT2T.Sample.SuperPopular.Thermostat.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://standards.freedesktop.org/dbus/introspect-1.0.dtd"[]>
+<node>
+  <interface name="org.OpenT2T.Sample.SuperPopular.Thermostat">
+  
+    <!-- Turn on the HVAC -->
+    <method name="turnOn"/>
+    
+     <!-- Turn off the HVAC -->
+    <method name="turnOff"/>
+    
+    <!-- Get the current temperature -->
+    <method name="getCurrentTemperature">
+      <arg name="temp" type="d" direction="out" />
+    </method>
+   
+   <!-- set the current temperature -->
+    <method name="setCurrentTemperature">
```

Comment:
This should be named something different, since setting a temperature on the thermostat does not set the "current" temperature, that is the ambient temperature measured by the thermostat. I think the term "setpoint" is most often used for setting the desired temperature on a thermostat. And the schema should support retrieving the setpoint separately from the current temperature.

 Also, most thermostats support separate setpoints for heating and cooling.